cli/src/cli_util.rs

```diff
@@ -3062,6 +3063,102 @@ impl fmt::Display for RemoteBookmarkNamePattern {
     }
 }
 
+/// Computes the location (new parents and new children) to place commits.
+///
+/// The `destination` argument is mutually exclusive to the `insert_after` and
+/// `insert_before` arguments.
+pub fn compute_commit_location(
+    ui: &mut Ui,
+    workspace_command: &mut WorkspaceCommandHelper,
+    destination: &Option<Vec<RevisionArg>>,
+    insert_after: &Option<Vec<RevisionArg>>,
+    insert_before: &Option<Vec<RevisionArg>>,
```

Comment:
nit: `Option<&[..]>` is more common as arguments. It might be better to add named struct.